Automattic/jetpack
# Jetpack: show Jetpack AI product in the plan products list

Fixes https://github.com/Automattic/jetpack/issues/31272

## Proposed changes:

* Jetpack: show Jetpack AI product in the plan products list

## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Get the Jetpack AI product
* Go to the Jetpack / My Plan page
* **Before**, confirm Jetpack AI plan is not in the products list
* **After**, confirm that the Jetpack AI product is listed now

<img width="859" alt="Screenshot 2023-06-09 at 08 10 36" src="https://github.com/Automattic/jetpack/assets/77539/af3ea62c-6478-4179-b317-e62efbd923de">
